chapter04/Task_4_3.py

```diff
@@ -0,0 +1,67 @@
+# Word Jumble
+#
+# The computer picks a random word and then "jumbles" it
+# The player has to guess the original word
+
+import random
+
+# create a sequence of words to choose from
+WORDS = ("python", "jumble", "easy", "difficult", "answer", "xylophone")
+# pick one word randomly from the sequence
+word = random.choice(WORDS)
+# create a variable to use later to see if the guess is correct
+correct = word
+# create tips
+TIPS = ("The language we learn to program in",
+        "What did we do with the letters of the words?!",
+        "What do we say when we win CS: GO?",
+        "The kid who got caught stealing - ",
+        "What do we want to get to the question - ",
+        "Strange musical instrument")
+# create score
+score = 100
+
+# create a jumbled version of the word
+jumble = ""
+while word:
+    position = random.randrange(len(word))
+    jumble += word[position]
+    word = word[:position] + word[(position + 1):]
+
+# start the game
+print(
+    """
+               Welcome to Word Jumble!
+    
+       Unscramble the letters to make a word.
+    (Press the enter key at the prompt to quit.)
+    """
+)
+print("The jumble is:", jumble)
+
+guess = input("\nYour guess: ")
+
+
+while guess != correct and guess != "":
+    if guess == "?":
+        score -= 25
+        if correct == (WORDS[0]):
```

Comment:
Instead of having this long "if", you could do something like that:
instead of "word = random.choice(WORDS)" you could have
correct_index = random.randint(0, len(WORDS) - 1)
correct = WORDS[correct_index]

Here you have an index, so later instead of "if" you could write
print(TIPS[correct_index])